src/Polynomials/MonomialBases.jl

```diff
@@ -406,6 +408,34 @@ function _gradient_nd!(
 
 end
 
+function _gradient_nd!(
+  v::AbstractVector{G},
+  x,
+  orders,
+  terms::AbstractVector{CartesianIndex{D}},
+  c::AbstractMatrix{T},
+  g::AbstractMatrix{T},
+  ::Type{V},
+  ::Val{true}) where {G,T,D,V}
```

Comment:
Even if it is not needed, for readability I would use here an intention revealing name. Something like Tisbitstype, not a big deal though.

Reply:
I refer to the name of the function parameter.

Reply:
done

Reply:
Thanks!